src/content/pages/earn/token-holder.md

```diff
@@ -2,4 +2,34 @@
 template: earn-page/token-holder
 path: /earn/token-holder
 title: Earn Token Holder Page
+tokenHolderInfo:
+  title: Token Holder
+  description: As a Token Holder, you can make the most of your tokens on the
+    Threshold Network by participating in DAO governance and voting on what’s
+    next for Threshold. Legacy NU or KEEP tokens should first be migrated to T.
+  image: /images/token-holder.png
+  buttons:
+    - label: Migrate to T
+      url: https://dashboard.threshold.network/
+      variant: EXTERNAL_SOLID
+  rowReverse: false
+secondaryInfo:
+  left:
+    label: T Token Holders
+    title: Participate in Threshold Governance!
+    description: As a T token holder, you can participate in the Threshold DAO and
+      vote to decide what's next for the network.
+    buttons:
+      - label: About Governance
+        url: /governance
+        variant: EXTERNAL_OUTLINE
```

Comment:
Should't be `INTERNAL_OUTLINE` because it is an internal link?

Reply:
great catch